### ISSUE TITLE
Loss exploded on multi-speakers training

### ISSUE THREAD

**Issue by sun-peach** [mozilla/LPCNet#113]
Hi, I am using the LPCNet to train vocoder on multiple speakers.

The procedure is training one model on 1st speaker for 5 epochs, and then use the "adaptation" in the code to train on the 2nd speaker for 5 epochs. Everything is good when I train the 1st speaker, loss goes to 2.7; but when I save the model and load it for next speaker (the 2nd speaker), the loss shows > 5.4 after the second epoch, and stay constant afterward.

I modify the code a little bit to incorporate the Keras "multi_gpu_model" function:
```
raw_model, _, _ = lpcnet.new_lpcnet_model(training=True)
if not adaptation: raw_model.summary()
if gpu_num >1:
    model = multi_gpu_model(raw_model, gpus=gpu_num)
else: model = raw_model

model.compile(optimizer='adam', loss='sparse_categorical_crossentropy', metrics=['sparse_categorical_accuracy'])
```

Could anyone tell me what is wrong with my training? Thank you very much!

**Comment by jmvalin** [comment]
If you read the README file, you'll see you can just concatenate all your speech from all speakers and train on the result. Don't train one epoch on one speaker and the next epoch on a different speaker. It's a lot of effort to achieve worse results.
